bindings/node/lib/autoEncrypter.js

```diff
@@ -10,40 +10,57 @@ module.exports = function(modules) {
   const cryptoCallbacks = require('./cryptoCallbacks');
 
   /**
-   * @typedef AutoEncrypter~AutoEncryptionExtraOptions
-   * @prop {string} [mongocryptdURI] overrides the uri used to connect to mongocryptd
-   * @prop {boolean} [mongocryptdBypassSpawn=false] if true, autoEncryption will not spawn a mongocryptd
-   * @prop {string} [mongocryptdSpawnPath] the path to the mongocryptd executable
-   * @prop {string[]} [mongocryptdSpawnArgs] command line arguments to pass to the mongocryptd executable
+   * Configuration options for a automatic client encryption.
+   *
+   * **NOTE**: Support for client side encryption is in beta. Backwards-breaking changes may be made before the final release.
```

Comment:
I think we probably want to remove this comment about being in "beta" since we're going GA